### PR TITLE
Logout page

### PR DESCRIPTION
**Pull request by sharathnair9999**
logout page

### CODE REVIEW

**Review by tewarimohit — APPROVED** [review]
Code looks good to me..

**Review by amarphule — APPROVED** [review]
looks great

File: styles/Header/header.css [hunk 1]
```diff
@@ -243,7 +243,7 @@ nav .nav-actions{
     height: 50vh;
     border-bottom-left-radius: 10px;
```
Comment (VedanthB) [inline styles/Header/header.css]: use rem here too

File: styles/Login/login.css [hunk 2]
```diff
@@ -55,4 +63,13 @@
   justify-content: flex-start;
   padding: 4px 0;
```
Comment (VedanthB) [inline styles/Login/login.css]: same here use rem

File: styles/Login/login.css [hunk 3]
```diff
@@ -55,4 +63,13 @@
...
-}
\ No newline at end of file
+}
+
+.logout-container .img-container{
+  max-width: 30rem;
+}
+
+.img-container img{
+  border-radius: 10px;
```
Comment (VedanthB) [inline styles/Login/login.css]: here too, use rem, be consistent with the units you use throughout your project

**Review by VedanthB — APPROVED** [review]
looks good to go 🚀

File: styles/Login/login.css (hunk 3 above)
Comment (sharathnair9999) [reply styles/Login/login.css]: Done.

**Pull request merged**